pkg/master/reconcilers/reconcilers.go

```diff
@@ -38,18 +38,32 @@ type EndpointReconciler interface {
 	ReconcileEndpoints(serviceName string, ip net.IP, endpointPorts []api.EndpointPort, reconcilePorts bool) error
 }
 
+// Type the reconciler type
+type Type string
```

Comment:
You may need to be more descriptive, like: `ReconcilerType`

Reply:
We ended up here because before it was stuttering (see "Avoid Stuttering" [here](https://blog.golang.org/package-names)).